CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * [ENHANCEMENT] Ruler: Added `-ruler.alertmanager-client.tls-enabled` configuration for alertmanager client. #3432 #3597
 * [ENHANCEMENT] Activity tracker logs now have `component=activity-tracker` label. #3556
+* [ENHANCEMENT] Query-frontend: track all HTTP requests in the Activity Tracker. #3561
```

Comment:
Didn't this say "query HTTP requests" before? :thinking:

Reply:
Oh yes, I lost that after rebasing multiple times because of the ongoing Mimir release. Will fix in the last rebase, since I'm already having conflicts in the changelog.

Reply:
Rebased and fixed now.